release-notes/major12/12.0.0/migration.md

```diff
@@ -211,6 +211,20 @@ This page describes how to update Valtimo from the previous version to the curre
   The new implementations of `KvKProvider` and `BsnProvider` (`ZaakKvkProvider` and `ZaakBsnProvider`) no longer depend on the `openzaak` module.
   Please make sure you have configured at least one Zaken API Plugin to make these providers work.
 
+* **Zgw case configuration**
```

Comment:
Nice, but could you also add under "New library: @valtimo/zgw" that ZgwModule needs to be imported in AppModule in order for Case detail synchronisation to show